scripts/context.js

```diff
@@ -72,18 +73,29 @@ function singlePageView() {
           chrome.storage.sync.set({selectedFeature: selectedFeature}, function() {
           })
         })
-        //Set the default feature to be opened
         chrome.storage.sync.get(['selectedFeature'], function(result) {
```

Comment:
Please take this out of this for loop and declare a `var openedFeature` there !
Thanks